hibike/unit_tests/hibike_message_tests.py

```diff
@@ -0,0 +1,386 @@
+"""
+Unit tests for functions in hibike_message.
+"""
+
+
+import unittest
+import random
+import queue
+import time
+import threading
+
+import serial
+
+import hibike_message
+import spawn_virtual_devices
+DEVICE_TYPES = list(hibike_message.DEVICES)
+
+
+def test_with_random_data(func, arg_func, kwarg_func=lambda: {}, times=5, *args, **kwargs):
+    """
+    Test FUNC with random arguments generated by ARG_FUNC and KWARG
+    func, repeating TIMES.
+    """
+    # Do not expand returned lists out
+    multiple_args = isinstance(arg_func(), tuple)
+    if multiple_args:
```

Comment:
get rid of the if-branch:
`multiple_args = tuple(arg_func())`

Or, just specify that arg_func must return an iterable in your docstring